scopesim/utils.py

```diff
@@ -863,11 +863,18 @@ def from_currsys(item):
     if isinstance(item, str) and len(item) and item.startswith("!"):
         if item in rc.__currsys__:
             item = rc.__currsys__[item]
+            if isinstance(item, str) and item.startswith("!"):
+                item = from_currsys(item)
```

Comment:
According to codecov, this case isn't covered by a unit test. Care to add one? 🙂

Reply:
done in latest commit